vllm/v1/worker/gpu_model_runner.py

```diff
@@ -1872,11 +1955,27 @@ def _dummy_run(
                 intermediate_tensors = self.sync_and_slice_intermediate_tensors(
                     num_tokens, None, False)
 
+            # Prepare dummy ALoRAMetadata
+            dummy_k_offsets = torch.tensor([1] * max_num_reqs,
+                                           device=self.device)
+            dummy_cu_num_tokens = np.cumsum(num_scheduled_tokens)
+            dummy_query_start_loc = [0] * (max_num_reqs + 1)
+            dummy_query_start_loc[0] = 0
+            dummy_query_start_loc[1:num_reqs + 1] = dummy_cu_num_tokens
+            dummy_query_start_loc = torch.tensor(dummy_query_start_loc,
+                                                 device=self.device)
```

Comment:
Don't we need to create a query_start_locs for the attention metadata above? Can't we just re-use that here?

Reply:
Done